FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -279,11 +279,13 @@ extension Auth: AuthInterop {
   /// - Parameter completion: Optionally; a block which is invoked when the list of sign in methods
   /// for the specified email address is ready or an error was encountered. Invoked asynchronously
   /// on the main thread in the future.
-  @available(
-    *,
-    deprecated,
-    message: "`fetchSignInMethods` is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled."
-  )
+  #if !FIREBASE_CI
+    @available(
+      *,
+      deprecated,
+      message: "`fetchSignInMethods` is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled."
+    )
+  #endif // !FIREBASE_CI
```

Comment:
```suggestion
  @available(
    *,
    deprecated,
    message: "`fetchSignInMethods` is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled."
  )
```

Reply:
Will commit and expect that pod lib lint CI will fail

Reply:
As expected, CI failed: https://github.com/firebase/firebase-ios-sdk/actions/runs/9881370223/job/27292176682?pr=13264